README.md

```diff
@@ -410,6 +410,17 @@ Old method(s): is_valid_location_dest_id
 One of [`location_id`, `location_name`, `location_barcode`] must
 be specified in the request.
 
+### Create stock investigation
+```
+URI: /api/stock-picking-batch/:ident/unpickable
+HTTP Method: POST
+Old method(s): unpickable_item
+```
+* @param ident - id of the batch to mark as having an unpickable move line
+* @param move_line_id - id of the move_line that is unpickable
+For the given batch_id and move_line_id will generate a stock investigation picking for the relevant stock move lines.
```

Comment:
This looks like the description and it is merged with the description of the move_line_id paramater. Move it before line 419.